cli/args_test.go

```diff
@@ -147,7 +147,11 @@ func assertOptionsEqual(t *testing.T, expected options.TerragruntOptions, actual
 }
 
 func mockOptions(terragruntConfigPath string, workingDir string, terraformCliArgs []string, nonInteractive bool, terragruntSource string, ignoreDependencyErrors bool) *options.TerragruntOptions {
-	opts := options.NewTerragruntOptionsForTest(terragruntConfigPath)
+	logger := util.CreateLogger("")
+	opts, err := options.NewTerragruntOptionsForTest(terragruntConfigPath)
+	if err != nil {
+		logger.Panicf("error: %v\n", errors.WithStackTrace(err))
```

Comment:
I know you're trying to minimize yak shaving, but calling `panic` during tests is still problematic. I'd recommend adding a `t *testing.T` param to this test and calling `t.FatalF` here if you get an error.